docs/changelog.rst

```diff
@@ -8,6 +8,19 @@ KSQL 5.4.0 includes new features, including:
 
 * UDAFs support STRUCTs as parameters and return values.
 
+* KSQL now supports working with source data where the value is an anonymous Avro serialized
+  `ARRAY`, `MAP` or primitive type, for example `STRING` or `BIGINT`. Previously KSQL required all
+  Avro values to be a record.  For more information, refer to :ref:`ksql_single_field_wrapping`.
+
+* KSQL now supports working with source data where the value is an anonymous JSON serialized
```

Comment:
Maybe combine with the previous paragraph? "KSQL now supports working with source data where the value is an anonymous Avro or JSON serialized..."